src/components/CategoryPage.jsx

```diff
@@ -1,9 +1,19 @@
 import React from 'react';
+import {useSelector, useDispatch} from 'react-redux';
+import {checkStatus} from '../redux/categories/categories'
 
-const CategoryPage = () => (
-  <div>
-    <button type="button">CHECK STATUS</button>
-  </div>
-);
+const CategoryPage = () => {
+  const categories = useSelector((state) => state.checkReducer.categories);
```

Comment:
- [ ] Nothing is being displayed on your categories page. If you visit the page and check your console, you will see this is due to some errors here. You need to change this line to the code below as what you have in your store is `category`, not `checkReducer.categories`.

  ```
  const categories = useSelector((state) => state.category);
  ```

- [ ] After that you need to refactor your categories file in the redux to this.

  ```
  const CHECK_STATUS = 'bookStore/categories/CHECK_STATUS';

  const categoriesState = [];
  export const checkStatus = () => ({
    type: CHECK_STATUS,
  });
  
  const checkReducer = (state = categoriesState, action) => {
    switch (action.type) {
      case CHECK_STATUS:
        return 'Under construction...';
      default:
        return state;
    }
  };
  
  export default checkReducer;
  ```